test-app/webpack.config.js

```diff
@@ -37,6 +40,20 @@ module.exports = (environment) => {
       filename: '[name].js',
       path: path.resolve(__dirname, 'dist'),
     },
-    plugins: [...htmlPlugins],
+    plugins: [
+      ...htmlPlugins,
+      new CopyPlugin({
+        patterns: [
+          {
+            from: 'assets/images/**/*',
+            context: path.resolve(
+              __dirname,
+              'node_modules/@lblod/embeddable-say-editor/dist'
+            ),
+            to: path.resolve(__dirname, 'dist'),
+          },
+        ],
+      }),
+    ],
```

Comment:
Not sure if it's really ideal to require from our users to add this to the webpack config of their application, unsure if there is a better way though.